packages/primitives/tooltip/src/tooltip.config.ts

```diff
@@ -0,0 +1,22 @@
+import { inject, InjectionToken, Provider } from '@angular/core';
+import { RdxTooltipConfig } from './tooltip.types';
+
+export const defaultTooltipConfig: RdxTooltipConfig = {
+    delayDuration: 700,
+    skipDelayDuration: 300
+};
+
+export const RdxTooltipConfigToken = new InjectionToken<RdxTooltipConfig>('RdxTooltipConfigToken');
+
+export function provideRdxTooltipConfig(config: Partial<RdxTooltipConfig>): Provider[] {
+    return [
+        {
+            provide: RdxTooltipConfigToken,
+            useValue: { ...defaultTooltipConfig, ...config }
+        }
+    ];
+}
+
+export function injectTooltipConfig(): RdxTooltipConfig {
+    return inject(RdxTooltipConfigToken, { optional: true }) ?? defaultTooltipConfig;
```

Comment:
you can pass `defaultTooltipConfig` as the second parameter, as the default value

```ts
export const RdxTooltipConfigToken = new InjectionToken<RdxTooltipConfig>(
    'RdxTooltipConfigToken', 
    { factory: () => defaultTooltipConfig }
);
```

then:
```suggestion
    return inject(RdxTooltipConfigToken);
```

Reply:
The provideRdxTooltipConfig and injectTooltipConfig approach is more flexible, maintainable, and scalable for real-world use cases. 

It better supports dynamic configuration, testing, and modularity, making it the preferred solution over using the InjectionToken factory alone.

Reply:
I suggest keep the current solution! reasons described by @pimenovoleg